### PR TITLE
Feature/ea/add button counter

### PR DESCRIPTION
**Pull request by andres-acc**
## Summary
* Improve existing components, button and footer
* Add badge to button component
* Map results counter from api

## Preview
ezgif com-video-to-gif (5)

## Jira Card
https://trello.com/c/xpMvpmbO

### CODE REVIEW

File: src/app/components/button/button.component.ts [hunk 1]
```diff
@@ -7,6 +7,7 @@ import { Component, EventEmitter, Input, Output } from '@angular/core';
 })
 export class ButtonComponent  {
```
Comment (YgRamos9) [inline src/app/components/button/button.component.ts:8]: - format please 

```suggestion
export class ButtonComponent {
```

File: src/app/pages/search-character/search-character.component.html [hunk 2]
```diff
@@ -9,8 +9,9 @@ <h1 class="title">Encuentra a tu personaje favorito</h1>
       (onGenderSelectorChange)="handleGenderChange($event)"
       (onSpecieSelectorChange)="handleSpecieChange($event)"
     ></app-big-input>
-    <ng-container *ngIf="charactersList.length > 0">
+    <div class="results" *ngIf="charactersList.length > 0">
```
Comment (YgRamos9) [inline src/app/pages/search-character/search-character.component.html:12]: - This is where I mention the conditional that would not be necessary for the counter button

**Review by YgRamos9 — APPROVED** [review]

File: src/app/components/button/button.component.html [hunk 3]
```diff
@@ -1 +1,7 @@
-<button class="button" [title]="content" (click)="buttonAction()">{{ content }}</button>
+<button 
+  class="button" 
+  [title]="content" 
+  (click)="buttonAction()">
+    {{ content }}
+    <span class="badge">{{badge}}</span>
+</button>
```
Comment (miguelhenaoa) [inline src/app/components/button/button.component.html:7]:
```suggestion
<button 
  class="button" 
  [title]="content" 
  (click)="buttonAction()"
>
    {{ content }}
    <span class="badge">{{badge}}</span>
</button>
```

File: src/app/components/footer/footer.component.scss [hunk 4]
```diff
@@ -22,21 +22,17 @@
   }
 
   .floating-image {
+    bottom: 15px;
     background-repeat: no-repeat;
     position: absolute;
```
Comment (miguelhenaoa) [inline src/app/components/footer/footer.component.scss:27]: Sort

**Review by miguelhenaoa — APPROVED** [review]

**Pull request merged**